code/core/src/core-server/presets/common-manager.ts

```diff
@@ -3,7 +3,6 @@ import { global } from '@storybook/global';
 
 import { addons } from 'storybook/manager-api';
 
-/* eslint-disable prettier/prettier */
```

Comment:
This is questionable. To make sure prettier doesn't mess with our sensitive files in forked repos, we need to actually add them to prettierignore.

But this is a less good setup for us where we know internally that specific bits of the files should be ignored, but still want automatic formatting on the rest of the files through ESLint.

Reply:
Why is prettierignore better for forks here?

Is it because forks run prettier directly instead of via eslint? If so, can we just add a prettierignore comment as well as the existing eslint comment?

Reply:
(btw I meant that my code change was questionable, not the original code ;-) so thanks for questioning it!)